README.md

```diff
@@ -137,6 +137,54 @@ There are 2 ways to run the server: with or without `autoexport`. When the app r
 
 In order to be able to test your localhost from a VM (such as VirtualBox) you'll have to add your IPv4 address to the server configuration. You can read how it can be done [here](docs/what-if/use-custom-host.md)
 
+## Valtech branching strategy
```

Comment:
Might be worthy of it's own readme file / what if file, just so the top level read me can be kept to a clean minimum?

Reply:
Hi @jacsamell This little section is only there to give the developer a reference to an external file. We always add only short sections of important information to the README, and move the lengthy stuff to 'what-ifs'

Reply:
I meant the section below really Steps 1,2,3,4,5 maybe that could be moved to an external document. I'm not too fussed on this, people can just ignore that section if they want 🙂